scalding-core/src/main/scala/com/twitter/scalding/ExecutionUtil.scala

```diff
@@ -2,17 +2,27 @@ package com.twitter.scalding
 
 import com.twitter.algebird.Semigroup
 
+import scala.util.{Failure, Success, Try}
+
 object ExecutionUtil {
   /**
+   * Useful for when using parallelism and want to output the final state of the job
+   */
+  def liftedSequenceUnit[T](executions: Execution[Seq[Try[T]]], handleComplete: Try[T] => Unit): Execution[Unit] =
+    executions.onComplete {
+      case Success(sequence) => sequence.foreach(handleComplete)
+      case Failure(ex) => () //If the execution was lifted this should never occur
+    }.unit
+
+  /**
    * Generate a list of executions from a date range
    *
    * @param duration Duration to split daterange
-   * @param parallelism How many jobs to run in parallel
    * @param fn Function to run a execution given a date range
    * @return Sequence of Executions per Day
    */
-  def executionsFromDates[T](duration: Duration, parallelism: Int = 1)(fn: DateRange => Execution[T])(implicit dr: DateRange): Seq[Execution[T]] =
-    dr.each(duration).map(fn).toSeq
+  def executionsFromDates[T](duration: Duration)(fn: DateRange => Execution[T])(implicit dr: DateRange): Seq[Execution[(DateRange, T)]] =
```

Comment:
Yea I worried about that, wasn't sure if this made it out on a release or not.  Thought the release was held up by some dependency problems.  

Let me refactor a bit to avoid breaking the API.